release.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import os
+import zipfile
+import argparse
+
+def zipall(ob, path, rel=""):
+    basename = os.path.basename(path)
+    if os.path.isdir(path):
+        if rel == "":
+            rel = basename
+        ob.write(path, os.path.join(rel))
+        for root, dirs, files in os.walk(path):
+            for d in dirs:
+                zipall(ob, os.path.join(root, d), os.path.join(rel, d))
+            for f in files:
+                ob.write(os.path.join(root, f), os.path.join(rel, f))
+            break
+    elif os.path.isfile(path):
+        ob.write(path, os.path.join(rel, basename))
+    else:
+        pass
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Release a new version')
+    parser.add_argument('version', type=str, help='version (e.g., 0.1.0)')
+    parser.add_argument('--source', dest='source', default=os.getcwd(),
+                        type=str, help='source folder')
+    parser.add_argument('--filename', dest='filename', default='PageNumberingAddon',
+                        type=str, help='filename')
+    
+    args = parser.parse_args()
+
+    release = args.version.replace('.', '_')
+    release_dir = os.path.join(
+        args.source,
+        'LibreOffice/versions/{}'.format(release)
+    )
+
+    os.makedirs(release_dir, exist_ok=True)
+
+    files_dir = os.path.join(
+        args.source,
+        'OpenOffice/python/oxt_metadata'
```

Comment:
Please change this to LibreOffice

Reply:
I take it form [`master`](https://github.com/arvchristos/lo-page-numbering/tree/master/OpenOffice/python/oxt_metadata) which is currently "OpenOffice/"

Reply:
I see. There are two versions of this extension, one for LibreOffice and one for OpenOffice, mainly because there are slight differences at each development process. 

I suggested you change the two argument prefixes to LibreOffice while I should have mentioned OO too. One idea is to include another argument -L or -O to select whether to export for OpenOffice or LibreOffice.